flow360/component/flow360_params/solvers.py

```diff
@@ -559,16 +563,21 @@ def update_model(self):
             "DDES": self.DDES,
             "gridSizeForLES": self.grid_size_for_LES,
             "quadraticConstitutiveRelation": self.quadratic_constitutive_relation,
-            "reconstructionGradientLimiter": self.reconstruction_gradient_limiter,
             "modelConstants": self.model_constants,
         }
 
         try_set(model, "rotationCorrection", self.rotation_correction)
+        
+        if self.reconstruction_gradient_limiter is not None:
+            model["reconstructionGradientLimiter"] = self.reconstruction_gradient_limiter
```

Comment:
It is still not ideal in the sense that not all default values are set by the SA/KOmega.